### PR TITLE
Add YAML schedule for extra_tests_in_textmode

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
- Related ticket: https://progress.opensuse.org/issues/57290
- Verification run: http://openqa.slindomansilla-vm.qa.suse.de/tests/2094

### CODE REVIEW

File: schedule/functional/extra_tests_in_textmode.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - console/lshw
+    - console/kmod
+    - console/suse_module_tools
+    - console/zziplib
+    - console/firewalld
+    - console/aaa_base
+    - console/osinfo_db
+    - console/orphaned_packages_check
+    - console/zypper_ref
+    - console/docker # should go to extra_tests_in_textmode_cont
```
Comment (foursixnine) [inline schedule/functional/extra_tests_in_textmode.yaml]: containers*

File: schedule/functional/extra_tests_in_textmode.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    - console/dstat
+    - x11/evolution/evolution_prepare_servers
+    - console/mutt
+    - console/supportutils
+    - console/mdadm
+    - console/journalctl
+    - console/quota
+    - console/vhostmd
+    - console/rpcbind
+    - sysauth/sssd # should go to extra_tests_in_textmode_phub
```
Comment (foursixnine) [inline schedule/functional/extra_tests_in_textmode.yaml]: :+1:

File: schedule/functional/extra_tests_in_textmode.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    - console/ca_certificates_mozilla
+    - console/unzip
+    - console/machinery # should go to extra_tests_in_textmode_phub
+    - console/gpg
+    - console/rsync
+    - console/clamav
+    - console/shells
+    - console/sudo
+    - console/dstat
+    - x11/evolution/evolution_prepare_servers
```
Comment (foursixnine) [inline schedule/functional/extra_tests_in_textmode.yaml]: Is it really needed here? or it's just a test that's there to support mutt? (I.e, setting up a mailserver?)

File: schedule/functional/extra_tests_in_textmode.yaml (hunk 1 above)
Comment (SergioAtSUSE) [reply schedule/functional/extra_tests_in_textmode.yaml]: I am following our own abbreviations: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/registration.pm#L387

Comment (SergioAtSUSE) [reply schedule/functional/extra_tests_in_textmode.yaml]: ok, it should be contm if we decide to use abbreviation

File: schedule/functional/extra_tests_in_textmode.yaml (hunk 3 above)
Comment (SergioAtSUSE) [reply schedule/functional/extra_tests_in_textmode.yaml]: No idea, I just copy-pasted the list of current scheduled modules by main.pm
I am open to unschedule any module.

File: schedule/functional/extra_tests_in_textmode.yaml (hunk 1 above)
Comment (foursixnine) [reply schedule/functional/extra_tests_in_textmode.yaml]: ok, makes sense... still... 

ew

**Comment by SergioAtSUSE** [comment]
Done in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9711

**Pull request closed without merging**